Readme.md

```diff
@@ -83,6 +83,32 @@ certbot plugins
 
 The resulting list should include `dns-porkbun` if everything went fine.
 
+#### Credentials file or cli parameters
+
+You can either use cli parameters to pass authentication information to certbot:
+
+```commandline
+...
+--dns-porkbun-key <your-porkbun-api-key> \
+--dns-porkbun-secret <your-porkbun-api-secret>
+```
+
+Or to prevent your credentials from showing up in your bash history, you can also create a credentials-file `porkbun.ini` (the name does not matter) with the following content:
+
+```ini
+dns_porkbun_key=<your-porkbun-api-key>
+dns_porkbun_secret=<your-porkbun-api-secret>
+```
+
+And then instead of using the `--dns-porkbun-key` parameter above you can use
```

Comment:
It should be added that the `--dns-porkbun-secret` program argument does not need to be used then either. This becomes indirectly clear by the following explanation with the priority of the arguments. But it would be better to state this explicitly in this case.

Reply:
Updated.